flight_company.rb

```diff
@@ -66,12 +70,71 @@
   else
     puts '=> Insira uma data válida e no formato dd/mm/aaaa.'
   end
-end if response_type.include?(arrival)
-
-puts ''
-puts '----------------------------'
-puts '   Resumo'
-puts " Aeroporto de origem: #{origin_airport.upcase}"
-puts " Aeroporto de destino: #{destination_airport.upcase}"
-puts " Data de partida: #{departure_time}"
-puts " Data de retorno: #{arrival_time.empty? ? 'sem data' : arrival_time }"
+end if ARRIVAL_RESPONSE.include?(arrival)
+
+def http_request(url)
+  http = Net::HTTP.new(url.host, url.port)
+  http.use_ssl = true
+
+  request = Net::HTTP::Get.new(url)
+  request["x-rapidapi-key"] = ENV['KEY_API']
```

Comment:
Aqui você está chamando de `KEY_API`, porém  no `.env.example` você chamou de  `TOKEN_API`, quando a pessoa for tentar rodar vai dar erro, não?

Reply:
:grimacing: Esqueci de ajustar também no `.env.example`.

Reply:
Feito em: https://github.com/Laryssa-lfa/flight-company/pull/2/commits/cc0be83e677e4e03d5f0c52c9217c540a55466e4